README.md

```diff
@@ -1,15 +1,31 @@
 # Smooth Checkout
 All the things you need during a Buildkite checkout :butter: :kite:
 
-## Repository-less builds
+## Usage
+
+### Repository-less builds
 ```yml
 steps:
   - command: echo "Skips checking out Git project in checkout" 
     plugins:
-      - hasura/smooth-checkout#v1.0.0:
+      - hasura/smooth-checkout#v1.1.0:
           skip_checkout: true
 ```
 
+### Checking out repo
+```yml
+steps:
+  - command: echo "Checks out repo at given ref"
+    plugins:
+      - hasura/smooth-checkout#v1.1.0:
+          clone_url: https://github.com/<username>/<reponame>
+          ref: <ref>
+```
+Allowed values for `ref`:
+- Branch name
+- Git tag
+- Commit SHA(40 character long hash)
```

Comment:
I tried the fetch command with short hash but it didn't work
![Screenshot from 2021-08-05 15-06-49](https://user-images.githubusercontent.com/32202683/128328854-8303d43a-f00d-4d5b-9ea0-2bceebe2f1a0.png)

Reply:
I was going throught the git manual for `git-fetch` and I found this statement
> \<src\> is typically a ref, but it can also be a fully spelled hex object name.

So I don't think short hashes are supported in `git-fetch`.
Reference: https://git-scm.com/docs/git-fetch#Documentation/git-fetch.txt-ltrefspecgt